kombu/messaging.py

```diff
@@ -115,7 +115,8 @@ def publish(self, body, routing_key=None, delivery_mode=None,
                 mandatory=False, immediate=False, priority=0,
                 content_type=None, content_encoding=None, serializer=None,
                 headers=None, compression=None, exchange=None, retry=False,
-                retry_policy=None, declare=None, expiration=None,
+                retry_policy=None, declare=None, expiration=None, timeout=None,
+                confirm_timeout=None,
```

Comment:
Not sure if this is right way how to pass confirm_timeout because:
1. it's used only when confirmation_mode is enabled
2. it's applicable only for py-amqp library.

In any case do we need this change since timeout is also passed if confirm_timeout=None?

https://github.com/celery/py-amqp/blob/a46054e5a30ad41f982708f3a6b5e9c02b123067/amqp/channel.py#L1803-L1807

Reply:
Yes, I agree with you.
PR changed and deleted confirm_timeout.

Reply:
@matusvalo